mk/xenadmin-build.sh

```diff
@@ -133,6 +133,10 @@ cd ${REPO}/XenAdminTests/bin/ && tar -czf XenAdminTests.tgz ./Release
 cd ${REPO}/XenAdmin/TestResources && tar -cf ${OUTPUT_DIR}/XenCenterTestResources.tar *
 cp ${REPO}/XenAdminTests/bin/XenAdminTests.tgz ${OUTPUT_DIR}/XenAdminTests.tgz
 
+#include cfu validator binary in output directory
+cd ${REPO}/XenAdmin/CFUValidator && zip -r CFUValidator.zip ./bin/Debug/CFUValidator
```

Comment:
We are building in Release mode, so you should zip the Release folder. 
The CFUValidator is not inside the XenAdmin folder, so you need `cd ${REPO}/CFUValidator`